content/blog/my-new-post/index.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "My New Post"
```

Comment:
This `my-new-post` directory should be deleted.

Reply:
deleted

Reply:
Still need to delete this dir.